source/base/Profiler.ooc

```diff
@@ -65,6 +70,8 @@ Profiler: class {
 	}
 	resetAll: static func { This _profilers apply(func (profiler: This) { profiler reset() }) }
 	dispose: static func {
```

Comment:
Agreed, it is one of the bad things coming from making `this` and `This` optional.